spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/security/CognitoAuthenticationAutoConfiguration.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.autoconfigure.security;
```

Comment:
Shouldn't the package be `io.awspring.cloud.autoconfigure.cognito`? In the same way, we do not have `io.awspring.cloud.resource` for S3.